Packs/Respond/IncidentFields/incidentfield-URL.json

```diff
@@ -1,18 +1,29 @@
 {
- "associatedTypes": [
-  "Respond Software Incident"
- ],
- "caseInsensitive": true,
- "cliName": "url",
- "editForm": true,
- "id": "incident_url",
- "name": "URL",
- "threshold": 72,
- "type": "shortText",
- "version": -1,
- "fromVersion": "5.0.0",
- "content": true,
- "system": false,
- "required": false,
- "group": 0
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Respond Software Incident"
+    ],
+    "caseInsensitive": true,
+    "cliName": "url",
```

Comment:
Please add a prefix to *all* custom incident fields.
You should use something like `respond-<cli_name>`.